src/miner.cpp

```diff
@@ -917,22 +917,22 @@ void ThreadStaker::operator()(std::vector<ThreadStaker::Args> args, CChainParams
             }
             static std::atomic<uint64_t> time{0};
             if (GetSystemTimeInSeconds() - time > 120) {
-                LogPrintf("ThreadStaker: unlock wallet to start minting...\n");
+                LogPrint(BCLog::STAKING, "ThreadStaker: unlock wallet to start minting...\n");
                 time = GetSystemTimeInSeconds();
             }
             std::this_thread::sleep_for(std::chrono::milliseconds(100));
         }
     }
 
-    LogPrintf("ThreadStaker: started.\n");
+    LogPrint(BCLog::STAKING, "ThreadStaker: started.\n");
```

Comment:
This is a single log message per staker. Better to leave an auditable log point.

Reply:
Will change it back